### PR TITLE
LWDEV-6276 stop-limit orders

### PR DESCRIPTION
**Pull request by barsukovskii**

### CODE REVIEW

File: src/main/kotlin/com/lykke/matching/engine/database/OrderBookDatabaseAccessor.kt [hunk 1]
```diff
@@ -6,4 +6,6 @@ import java.util.concurrent.PriorityBlockingQueue
 interface OrderBookDatabaseAccessor {
     fun loadLimitOrders(): List<NewLimitOrder>
     fun updateOrderBook(asset: String, isBuy: Boolean, orderBook: PriorityBlockingQueue<NewLimitOrder>)
+    fun loadStopLimitOrders(): List<NewLimitOrder>
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/database/OrderBookDatabaseAccessor.kt]: Separate OrderBook/StopOrderBook accessors

File: src/main/kotlin/com/lykke/matching/engine/database/file/FileOrderBookDatabaseAccessor.kt [hunk 2]
```diff
@@ -15,7 +15,7 @@ import java.util.LinkedList
 import java.util.concurrent.PriorityBlockingQueue
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/database/file/FileOrderBookDatabaseAccessor.kt]: The same, but add open class that will keep all staff related to work with dirs/files

File: src/main/kotlin/com/lykke/matching/engine/order/LimitOrderProcessor.kt [hunk 3]
```diff
@@ -0,0 +1,445 @@
+package com.lykke.matching.engine.order
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/order/LimitOrderProcessor.kt]: Separate LimitOrderProcessor and StopLimitOrderProcessor
Add generic LimitOrderProcessor that will encapsulate Limit/Stop processors and call them in proper way

File: src/main/kotlin/com/lykke/matching/engine/services/GenericLimitOrderService.kt [hunk 4]
```diff
@@ -9,6 +9,7 @@ import com.lykke.matching.engine.holders.AssetsHolder
 import com.lykke.matching.engine.holders.AssetsPairsHolder
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/services/GenericLimitOrderService.kt]: Separate Limit/Stop services

**Comment by barsukovskii** [comment]
@dmitrykrasovskih 
Did changes base on comments

**Pull request merged**